frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -62,6 +66,7 @@ const mapStateToProps = (state: RootState): StateProps => {
       .filter(
         (model: K8sKind) =>
           model.apiGroup === KNATIVE_SERVING_APIGROUP ||
+          model.apiGroup === KNATIVE_OPERATOR_APIGROUP ||
```

Comment:
I don't believe this is required because we don't display KnativeServing resources in the overview.
cc @invincibleJai since you two are doing the same work :P

Reply:
yeah, this is not needed as we get here knative services/revisions/configurations etc and their model will be `serving.knative.dev`